test/integration-stock/setup.js

```diff
@@ -39,6 +39,11 @@ before(() => {
   return agent.post('/auth/login').send(user);
 });
 
+beforeEach(done => {
+  this.timeout(10000);
```

Comment:
This isn't defined in an arrow function :wink:

You'll need to use `function () {}` for this.